Terminal.Gui/ConsoleDrivers/WindowsDriver.cs

```diff
@@ -891,22 +895,6 @@ void ProcessInput (WindowsConsole.InputRecord inputEvent)
 				}
 				break;
 
-			case WindowsConsole.EventType.WindowBufferSize:
-				var winSize = WinConsole.GetConsoleBufferWindow (out Point pos);
-				left = pos.X;
-				top = pos.Y;
-				cols = inputEvent.WindowBufferSizeEvent.size.X;
-				if (EnableConsoleScrolling) {
-					rows = Math.Max (inputEvent.WindowBufferSizeEvent.size.Y, rows);
-				} else {
-					rows = inputEvent.WindowBufferSizeEvent.size.Y;
-				}
-				//System.Diagnostics.Debug.WriteLine ($"{EnableConsoleScrolling},{cols},{rows}");
-				ResizeScreen ();
-				UpdateOffScreen ();
-				TerminalResized?.Invoke ();
-				break;
```

Comment:
But do you are remove the `EnableConsoleScrolling` feature? If you won't then it doesn't work without the code you deleted.

Reply:
The scroll bar isn't show on resize when `EnableConsoleScrolling` is enabled.

![scroll-without-scrolbar-cmd](https://github.com/gui-cs/Terminal.Gui/assets/13117724/f5da6f41-41f7-4dab-a029-3d8f538905db)

Reply:
The scroll back doesn't show as WT, showing all, but only showing the app data.

Reply:
See the difference:

![scroll-without-scrolbar-cmd](https://github.com/gui-cs/Terminal.Gui/assets/13117724/b0dc01eb-4d2e-4f09-869e-089ed096f69c)

Reply:
```cs
			ScreenBuffer = CreateConsoleScreenBuffer (
				DesiredAccess.GenericRead | DesiredAccess.GenericWrite,
				ShareMode.FileShareRead | ShareMode.FileShareWrite,
				IntPtr.Zero,
				1,
				IntPtr.Zero
			);

```

On WT `ScreenBuffer` doesn't create a separate buffer but only is using the WT's buffer. That why when we scroll back we see all mixed, terminal existed buffer and app buffer. On `cmd` or `conhost` a new buffer is created and that why only the app buffer is scrolled. I know you was always confused about that and I hope you understand it now.
But I understand you may don't want to use this feature anymore due due WT incompatibility and `Linux` not supporting this for now. Thanks.

Reply:
You're right that this confuses me. My apologies for that!

Reply:
@bdidsp, the latest in this PR:

- Seems to work wonderfully with WT. 
  - There's no need for EnableConsoleScrolling (scrolling works even without that flag enabled)
  - Resizing no longer sometimes leaves the screen in the confused state (I didn't mention this above)
  - The back buffer is mostly preserved (sometimes artifacts are left, but if the terminal is never resized it works as expected).
 -  On Conhost, things work fine except that EnableConsoleScrolling is broken. 

If you can suggest a change that keeps the WT behavior but fixes Conhost I'd love that!

Reply:
> @bdidsp, the latest in this PR:
> 
>     * Seems to work wonderfully with WT.
>       
Do you mean this what I'm showing in the gif bellow? Why do you want the scroll bar here? To allow to see the back buffer outside the app instead of only within the app? The app needs is own buffer and restore the original on exit. I think you really didn't understand what the `EnableConsoleScrolling` is for, sorry. I really like a lot WT but only don't understand why they don't allow an app create is own buffer. They explained for not to mixed to others tabs, but I think it isn't an excuse.

![scroll-without-scrolbar-wt](https://github.com/gui-cs/Terminal.Gui/assets/13117724/5835014a-2a8a-4c36-a653-04a880c4e227)

>       * There's no need for EnableConsoleScrolling (scrolling works even without that flag enabled)

Because you really don't understand what the `EnableConsoleScrolling` is for, sorry.

>       * Resizing no longer sometimes leaves the screen in the confused state (I didn't mention this above)

Really? I wouldn't like that my app leaves so much trash on exit :-)

![scroll-without-scrolbar-wt](https://github.com/gui-cs/Terminal.Gui/assets/13117724/08b3cb47-e8ce-48b9-91c1-06748b0b53cf)

>       * The back buffer is mostly preserved (sometimes artifacts are left, but if the terminal is never resized it works as expected).
> 
Well I don't think the back buffer was mostly preserved, you added more trash :-)
I already saw that after exit the app and resizing the terminal the buffer is messing, but that is a terminal responsibility or the app isn't restoring well. But I would really have avoided the mess of the app.

>     * On Conhost, things work fine except that EnableConsoleScrolling is broken.
> 
No it's working as expected without showing the scroll bar. `EnableConsoleScrolling` is broken because you removed the code for it working.

> 
> If you can suggest a change that keeps the WT behavior but fixes Conhost I'd love that!

WT as is, will always leaving mess on the back buffer within the app. A possible solution is detecting that WT is the terminal used and don't allow `EnableConsoleScrolling` be setting or not setting it at all. Even you don't enable `EnableConsoleScrolling`, by resizing the terminal it will leave messed the back buffer. I don't see that removing it will fix anything.
The behavior with the `Conhost` is because you removed all the `EnableConsoleScrolling` related code. If you are talking about the `Conhost` habving the same mess as WT, then forget that :-)